MCLOUD-2981: Asses usefulness and necessity of option Verbosity (#7708)
magento/devdocs: src/cloud/env/variables-deploy.md

-  **Default**—_Not set_
-  **Version**—Magento 2.1.4 and later

> Enable or disable the [Symfony](https://symfony.com/doc/current/console/verbosity.html) debug verbosity level for `bin/magento` CLI commands executed from the deployment script. 

Reviewer: ```suggestion

Enable or disable the [Symfony](https://symfony.com/doc/current/console/verbosity.html) debug verbosity level for `bin/magento` CLI commands executed from the deployment script. 
```

> To change verbosity of the output from _successful_ `bin/magento` CLI commands, you must set the `MIN_LOGGING_LEVEL` to `debug`.

Reviewer: ```suggestion

To change verbosity of the output from _successful_ `bin/magento` CLI commands, you must set the `MIN_LOGGING_LEVEL` to `debug`.
```

> Choose the level of detail provided in the logs: `-v`, `-vv`, or `-vvv`.

Reviewer: ```suggestion
Use the following options to set the level of detail provided in the logs: `-v`, `-vv`, or `-vvv`.
```
